### DISCUSSION TITLE
Updating Ombi - ✗ There is currently no update path available.

### DISCUSSION THREAD

**Discussion by jarjarfinks** [tteck/Proxmox#2770] [Help Needed] [answered]
The error message: 
`✗ There is currently no update path available.`

Shows up when I try to use the script to update Ombi. 

I checked `/opt` for ombi and it's definitely there. Any idea what's going on?

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/blob/1ccb2de7f6d68e2c0f0b3e341c88ee26eb0aef5f/ct/ombi.sh#L55-L60

**Reply by jarjarfinks** [reply]
Yeah I read through the script, but that doesn't help me figure it out....The script technically should work based on the code, is it maybe a permissions error?

**Reply by tteck** [reply]
There is literally no update path available.

**Reply by tteck** [reply, marked as answer]
Try it now https://github.com/tteck/Proxmox/commit/98ee62bca03837bfad36a689359a0ea4035514b7

**Reply by jarjarfinks** [reply]
Fixed!
